scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -207,20 +205,42 @@ <h4 class="card-header">Case: {{case.display_name}}</h4>
   </div>
 {% endmacro %}
 
-{% macro roh_panel() %}
-  {% if case.chromograph_prefixes %}
-  <div class="d-flex p-2 ">
-    <div class="card panel-default" id="rohPanel">
-      <div class="panel-heading">Ideogram, UPD, ROH</div>
-      <br>
-      <div class="panel-body">
-        <svg id="roh_svg" width="1500" height="1250"></svg>
+
+<!-- Iterate through Individuals, if chromograph_files is presenet add a panel for images -->
+{% macro insert_multi_image_panel() %}
+  {% for i in case.individuals %}
+  {% if i.chromograph_images %}
+  {{ add_collapsable_panel(i.individual_id, i.display_name) }}
+  {% else %}
+    <a> {{i.individual_id}} images not configured</a>
```

Comment:
Would be better silent!
![Uploading Screenshot 2020-12-12 at 07.05.36.png…]()

Reply:
Fixed!